.chloggen/recommedded-resource-attributes.yaml

```diff
@@ -0,0 +1,22 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
+
+# The name of the component, or a single word describing the area of concern, (e.g. collector, target allocator, auto-instrumentation, opamp, github action)
+component: auto-instrumentation
+
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Update recommended resource attributes to match the [semantic conventions](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/non-normative/k8s-attributes.md)
```

Comment:
Please add more info about the exact changes that need to be done for the operator.

Also, provide some kind of recommendation regarding upgrading.

Reply:
the subtext has all changes - is that sufficient?

The subtext should also cover what you need to consider for upgrading.